messages/pt-br.json

```diff
@@ -0,0 +1,144 @@
+			"select_update_mode": "Selecione o dispositivo que agora está no modo DFU (Device Firmware Update). Deve aparecer como \"HaritoraXW-Update\".",
+			"flash": "Isso vai atualizar a versão de firmware do seu dispositivo."
+		},
+		"button": {
+			"download_only": "Baixar",
+			"check_version": "Verificar versão",
+			"set_update_mode": "Colocar no modo de atualização",
+			"select_update_mode": "Selecionar dispositivo no modo DFU",
+			"flash": "Atualizar firmware"
+		},
+		"progress": {
+			"status": "Status: {status}",
+			"progress": "Progresso: {progress}%"
+		},
+		"status": {
+			"waiting": "esperando por usuário...",
+			"checking_version": "verificando versão...",
+			"got_version": "achou versão {version} ({date})",
+			"selecting": "selecionando dispositivo em modo de atualização...",
+			"updating": "atualizando firmware... ({current}/{total} bytes, {progress}%)",
+			"completed": "Atualização de firmware completada com sucesso!",
+			"cancelled": "atualização de firmware cancelada.",
+			"connection_lost": "conexão com o dispositivo perdida durante atualização.",
+			"invalid_firmware": "O arquivo de firmware selecionado é inválido ou está corrompido.",
+			"setting_update_mode": "colocando dispositivo no modo de atualização...",
+			"set_update_mode": "dispositivo está no modo de atualização!",
+			"selecting_dfu": "selecionando dispostivo no modo DFU...",
+			"dfu_selected": "dispositivo no modo DFU selecionado e está preparado para atualização.",
+			"starting_update": "começando atualização de firmware...",
+			"firmware_completed": "atualização de firmware completa!",
+			"please_select": "por favor selecione um dispositivo e firmware primeiro",
+			"firmware_updater_not_initialized": "atualizador de firmware não foi inicializado",
+			"status": "Status: {status}"
+		},
+		"debug_log": "Registro de depuração:"
+	},
+	"toasts": {
+		"show_all_versions_description": "Mostrar todas as versões de firmware, incluindo as que não forem testadas completamente. Isto não é recomendado!",
+		"packet_send_delay_description": "Intervalo entre mandando packets ao dispositivo. Um intervalo maior manda dados mais devagar mas é mais provável de atualizar seu dispositivo com sucesso. Usando o valor de -1 desativa isso completamente.",
+		"download_only_description": "Baixa apenas o arquivo de firmware ao seu computador sem atualizar o rastreador. Útil para usuários avançados experimentarem.",
+		"web_bluetooth_not_supported": "Web Bluetooth não é suportado (ou está desativado globalmente) neste navegador. Por favor use Chrome, Edge ou outro navegador baseado em Chromium para atualizar o firmware do seu dispositivo.",
+		"demo_mode_description": "Ativar modo de demonstração para simular o processo de atualização sem atualizar seu dispositivo de verdade. Apenas para mostrar!",
+		"demo_enabled": "Modo de demonstração está ativado - todas ações são simuladas"
+	},
```

Comment:
`command_copied`, `command_copy_failed`, `manual_update_description` (renamed from `download_only_description`), and `os_unsupported` missing